aep/general/0135/aep.md.j2

```diff
@@ -4,15 +4,45 @@ In REST APIs, it is customary to make a `DELETE` request to a resource's URI
 (for example, `/v1/publishers/{publisher}/books/{book}`) in order to delete
 that resource.
 
-Resource-oriented design (AIP-121) honors this pattern through the `Delete`
-method. These RPCs accept the URI representing that resource and usually return
+Resource-oriented design ([AEP-121][aep-121]) honors this pattern through the `Delete`
+method. These operations accept the URI representing that resource and usually return
 an empty response.
 
 ## Guidance
 
 APIs **should** generally provide a delete method for resources unless it is
 not valuable for users to do so.
 
+The Delete method **should** succeed if and only if a resource was present and
+was successfully deleted. If the resource did not exist, the method **should**
+send a `404 Not found` (`NOT_FOUND`) error.
+
+### Requests
+
+{% tab oas -%}
+
+Delete methods are specified using the following pattern:
+
+{% sample 'delete.oas.yaml', 'paths' %}
+
+- The HTTP verb **must** be `DELETE`.
```

Comment:
Some of these bullet points are the same for both OAS and proto.  Should we factor those out, and only put IDL-specific guidance in the tabs?

Reply:
I removed
> - The HTTP verb **must** be `DELETE`.

from the proto section. This can straight from the Google AIP but doesn't make sense given we have a whole separate section for HTTP guidance.